src/posthog-core.ts

```diff
@@ -864,6 +864,28 @@ export class PostHog {
         return data
     }
 
+    /**
+     * Capture a page view event.
+     *
+     * This is a wrapper around capture and is
+     * equivalent to capture('$pageview', {'$current_url': <url>})
+     *
+     * ### Usage:
+     *
+     *    posthog.page('https://example.com')
+     *
+     * @param {String} url The URL of the page. This will be included in the properties as '$current_url'.
+     * @param {Object} [properties] A set of properties to include with the event you're sending. These describe the user who did the event or details about the event itself.
+     * @param {Object} [config] Optional configuration for this capture request.
+     * @param {String} [config.transport] Transport method for network request ('XHR' or 'sendBeacon').
+     * @param {Date} [config.timestamp] Timestamp is a Date object. If not set, it'll automatically be set to the current time.
+     */
+    page(url: string, properties?: Properties | null, options?: CaptureOptions): CaptureResult | undefined {
```

Comment:
we'll also need to update usage docs on posthog.com

and we should have a cypress test that covers pageview events (even implicitly) which we can duplicate to validate that `page` causes actual events to be sent

Reply:
> we'll also need to update usage docs on posthog.com
To confirm, is this at https://github.com/PostHog/posthog.com/blob/master/contents/docs/libraries/js/index.mdx?

> and we should have a cypress test that covers pageview events (even implicitly) which we can duplicate to validate that page causes actual events to be sent

Does this mean something similar to `posthog-js/cypress/e2e/capture.cy.ts`? I'll start on something and raise a PR.